language/stdlib/modules/Vector.move

```diff
@@ -1,41 +1,45 @@
 address 0x1 {
 
-// A variable-sized container that can hold both unrestricted types and resources.
+/// A variable-sized container that can hold both unrestricted types and resources.
 module Vector {
 
     const EINDEX_OUT_OF_BOUNDS: u64 = 0;
 
+    /// Create an empty vector.
     native public fun empty<Element>(): vector<Element>;
 
-    // Return the length of the vector.
+    /// Return the length of the vector.
     native public fun length<Element>(v: &vector<Element>): u64;
 
-    // Acquire an immutable reference to the ith element of the vector.
+    /// Acquire an immutable reference to the `i`th element of the vector `v`.
```

Comment:
Nit maybe add the same "aborts if i is out of bounds" comment here as in `borrow_mut`